selfdrive/car/volkswagen/carcontroller.py

```diff
@@ -10,7 +10,7 @@ class CarController():
   def __init__(self, dbc_name, CP, VM):
     self.apply_steer_last = 0
 
-    self.packer_pt = CANPacker(DBC[CP.carFingerprint]['pt'])
+    self.packer_pt = CANPacker("vw_mqb_2010")
```

Comment:
Put string in values.py and import?

Reply:
Changed to something friendly to adding PQ later.